src/validators/min-length.ts

```diff
@@ -0,0 +1,37 @@
+import { LengthArgs, parseLengthParams, isLengthValid } from './length';
+import { FieldValidationFunctionSync } from '../model';
```

Comment:
Could be a good idea to have common base code for min and max validations?

Reply:
There is already a file length.js that already contains some common goodies, I was tempted to extract more stuff but decided to stop, the reason why: if somebody hops into this code and takes it as reference to create his own validator it should be something readable (almost all the validators follow the same pattern), what do you think?